src/components/BreadcrumbNav/BreadcrumbNav.stories.tsx

```diff
@@ -8,7 +8,7 @@ export default {
   component: BreadcrumbNav,
   decorators: [
     (Story) => (
-      <div className="sfds">
+      <div className="sfds darkBgLock">
```

Comment:
Question: Is this something we only need in storybook?

Reply:
I'd say yes. when this component is used anywhere in the app, it always has a dark background

Reply:
Sounds like we should apply the `darkBgLock` class to the `BreadCrumbBar` component in our `BreadcrumbNav` instead of only adding it here. Maybe this is a good topic for Design Eng pairing  session?